jobs/aws/eks-anywhere-build-tooling/autoscaler-1-21-presubmits.yaml

```diff
@@ -11,14 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+################################################################################
+# IMPORTANT: This Prowjob was generated by running `make prowjobs -C templater`.
+# DO NOT EDIT THIS FILE BY HAND! Refer to docs/prowjobs.md for instructions on
+# how to add a new Prowjob or update an existing Prowjob.
+################################################################################
 
 presubmits:
   aws/eks-anywhere-build-tooling:
   - name: autoscaler-1-21-presubmit
     always_run: false
-    run_if_changed: "EKS_DISTRO_MINIMAL_BASE_PYTHON_TAG_FILE|^build/lib/.*|Common.mk|projects/kubernetes/autoscaler/.*"
-    cluster: "prow-presubmits-cluster"
+    run_if_changed: "EKS_DISTRO_MINIMAL_BASE_TAG_FILE|^build/lib/.*|Common.mk|projects/kubernetes/autoscaler/.*"
```

Comment:
Should this be python or minimal base?

Reply:
@markapruett I looked in the project repo and the make file mentions the base file is only the minimal base